packages/cli/lib/artifact-generator.js

```diff
@@ -93,27 +93,7 @@ module.exports = class ArtifactGenerator extends BaseGenerator {
     });
 
     if (generationStatus) {
```

Comment:
Ideally, I think we should `generationStatus` file by file now instead of a global flag. For example,

models/model-1.ts (success)
models/model-2.ts (skipped)

Reply:
Would it be ok if I made a follow up task for this -- I assume this is needed for `lb4 openapi` which can call multiple generators under the hood?